metrics.py

```diff
@@ -0,0 +1,124 @@
+'''
+    a script that measures the similarity between input
+    training text and output inferenced text using bleu and rouge scores/
+
+    BLEU focuses on precision: how much the words (and/or n-grams)
+    in the candidate model outputs appear in the human reference.
+
+    ROUGE focuses on recall: how much the words (and/or n-grams)
+    in the human references appear in the candidate model outputs.
+
```

Comment:
You should add in this description, how the program should be run, a generic way and an example.